### PR TITLE
add matomo tracking

### PR DESCRIPTION
**Pull request by achen2401**
https://www.pivotaltracker.com/story/show/183594028

### CODE REVIEW

File: patientsearch/src/index.html [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
+          g = d.createElement("script"),
+          s = d.getElementsByTagName("script")[0];
+        g.type = "text/javascript";
+        g.async = true;
+        g.defer = true;
+        g.src = u + "matomo.js";
+        s.parentNode.insertBefore(g, s);
+      })();
+    </script>
+    <!-- End Matomo Code -->
```
Comment (mcjustin) [inline patientsearch/src/index.html:32]: @achen2401 looks like you copied the js that I listed for Sie-Ce for the project website, but instead you want the js from here: https://www.pivotaltracker.com/story/show/183594028/comments/233910403

Same issue as the other codebase, which I'm sure you've realized by now :)

**Review by mcjustin — CHANGES_REQUESTED** [review]

File: patientsearch/src/index.html (hunk 1 above)
Comment (achen2401) [reply patientsearch/src/index.html]: thank you @mcjustin.  I fixed it.

File: patientsearch/src/index.html [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
+    />
+    <!-- Matomo -->
+    <script type="text/javascript">
+        var _paq = window._paq || [];
+        _paq.push(['trackPageView']);
+        _paq.push(['enableLinkTracking']);
+        _paq.push(["setUserId", ""]);
```
Comment (mcjustin) [inline patientsearch/src/index.html:17]: @achen2401  is there a user ID that you can populate?

Comment (achen2401) [reply patientsearch/src/index.html:17]: @mcjustin I am not able to easily  obtain user id from here.  Is it required?

Comment (mcjustin) [reply patientsearch/src/index.html:17]: Let's flag for @pbugni  @ivan-c  - we'll want this for other things too, sooner or later (eg event logs, DCW, etc). Over in COSRI you asked about getting it from the JWT token.

**Pull request closed without merging**